_includes/homepage/hero/components/hero-infobox-desktop.html

```diff
@@ -0,0 +1,24 @@
+<div
+  class="p16 is-hidden-mobile"
+  style="width: {{- section.hero.size -}}; background: {% if section.hero.backgroundColor == 'gray' %} #00000080 {% else %} {{- section.hero.backgroundColor -}} {% endif %};"
```

Comment:
For my understanding - inline styling is needed here because we need to refer to `section.hero.backgroundColor`?

Reply:
yeap precisely